hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileCreateRequest.java

```diff
@@ -94,6 +96,12 @@ public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
     KeyArgs keyArgs = createFileRequest.getKeyArgs();
 
     // Verify key name
+    if (keyArgs.getKeyName()
```

Comment:
nit: this can be moved to `OmUtils` similar to [validateKeyName](https://github.com/apache/ozone/blob/3dd6e29f2deb1d27aef2e82fcf9ef962e996cc03/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java#L596) and reused it here and in `OMKeyCreateRequest`?

Reply:
yup nice catch, moved to HddsClientUtils, and reuse!